lib/strategies/base.js

```diff
@@ -80,6 +80,10 @@ BaseStrategy.prototype._reformatInput = function () {
 
   input.value = formattedState.value;
   setSelection(input, formattedState.selection.start, formattedState.selection.end);
+
+  if (this._afterReformatInput) {
```

Comment:
Not sure what you mean. I think I am already doing what you suggest. `this._afterReformatInput` only gets called if it exists, and (currently), it only exists in the Android Chrome strategy.